src/components/IconButton/IconButton.style.ts

```diff
@@ -1,54 +1,46 @@
-import { rem } from 'polished';
-import { Theme } from '../../theme';
+import { buttonStyle, heightBasedOnSize } from '../Button/Button.style';
```

Comment:
![image](https://user-images.githubusercontent.com/6433679/110778725-6571cb00-826b-11eb-965e-87c1f11a614d.png)
something doesnt look right

Reply:
swatches are not applied i guess for icon here as it does in the button

Reply:
![image](https://user-images.githubusercontent.com/28539607/110800463-dec9e780-8284-11eb-8bd4-e3341ffa0ab6.png)